Source/GUI/CraftBrowser.cs

```diff
@@ -158,6 +158,11 @@ public static ELCraftBrowser Spawn (ELCraftType type, string profile, SelectFile
 			cb.showMergeOption = showMergeOption;
 			cb.OnBrowseCancelled = onCancel;
 			cb.OnFileSelected = onFileSelected;
+			cb.OnConfigNodeSelected =
+				delegate (ConfigNode n, LoadType t) {
+					Debug.Log($"[CraftBrowserDialog] OnConfigNodeSelected - " + cb.selectedEntry.fullFilePath);
+					onFileSelected(cb.selectedEntry.fullFilePath, t);
+				};
```

Comment:
Well, there goes multi-version support (I checked). It was nice while it lasted.